e2e_samples/parking_sensors/scripts/deploy_appservice.sh

```diff
@@ -0,0 +1,27 @@
+export appName=data-simulator-$DEPLOYMENT_ID
+
+arm_output=$(
+    az deployment group create \
+  --resource-group $resource_group_name \
+  --template-file "./infrastructure/modules/appservice.bicep" \
+  --parameters appName=$appName \
+  --output json
+)
+
+if [[ -z $arm_output ]]; then
+    echo >&2 "ARM deployment failed."
```

Comment:
also please use `log` function moving forward

Reply:
We should probably add a issue to resolve all of the "echo"s...I just used another echo as a wrong example. ;)